docs/deep-dives/data/preenrollment_bias.md

```diff
@@ -0,0 +1,116 @@
+
+Inferences for the [average treatment effect](https://en.wikipedia.org/wiki/Average_treatment_effect) (ATE) are most commonly calculated by computed the average (mean) in each treatment branch and then computing the difference. However, this can also be calculated using linear models.
+
+As an example, we can fit a model of the form:
+
+$$y_i = \beta_0 + \beta_t t_i$$
+
+where $t_i$ is the treatment indicator (0 if control, 1 if treated) for the $i$-th unit. Inferences on the ATE can be found from the $\beta_t$ parameter. The point estimate and confidence interval will be identical ([ref](https://www.refsmmat.com/courses/727/lecture-notes/linear-models.html#sec-ols-framework)) to the point estimate and confidence interval of the absolute difference in means between the branches. Computing the confidence intervals for the relative differences are more complex, but can be done using post-estimation marginal effects ([ref](https://stats.stackexchange.com/questions/646454/inferences-on-ratio-of-branch-means-in-randomized-experiment/646462#646462)).
+
+Using this framework, it's quite simple to extend our experiment analysis to account for pre-experiment data. We can simply include it as a covariate in the model. We can instead estimate:
+
+$$y_i = \beta_0 + \beta_t t_i + \beta_y z_i$$
+
+Where $z_i$ is the metric of interest ($y$) for the $i$-th unit as measured during the pre-experiment period. As before, we're interested in inferences to $\beta_t$, but now these inferences will be:
+
+1. Adjusted to account for pre-experiment information
+2. Benefit from an increase in precision.
+
+### Configuring covariate adjustment
+
+#### Adjusting a new metric for preenrollment bias
+
+To perform adjustment for a new metric, you can write/edit the [custom config](../jetstream/configuration.md#custom-experiment-configurations) to do 2 things: 1) configure your metric to be calculated over the preenrollment period (that is, perform the retrospective A/A test) and 2) configure the adjustment.
+
+To ensure that your metric is computed during the pre-enrollment period, simply add it to the desired period metric list:
+
+```toml
+preenrollment_weekly = [
+    'my_new_metric'
+]
+```
+
+To configure the adjustment, first designate that inferences on the mean are desired using linear models. Then, configure that statistic to adjust based on the period chosen above. For example:
+
+```toml
+[metrics.my_new_metric.statistics.linear_model_mean] # desire to estimate the mean using linear models
+[metrics.my_new_metric.statistics.linear_model_mean.covariate_adjustment] # desire to adjust that estimate
+period = "preenrollment_week" # adjust using the same metric calculated during the week prior to enrollment
+```
+
+One can reference how adjustment is configured for guardrails ([example](https://github.com/mozilla/metric-hub/blob/main/jetstream/defaults/firefox_desktop.toml#L33-L67)).
```

Comment:
Something I always think about when linking to github code lines... Might be better to change these links from `blob/main` to `blob/<hash>` so that the links are still pointing to what we expect if the code changes.